examples/kubeflow/mnist_with_tfjob/README.md

```diff
@@ -0,0 +1,194 @@
+          containers:
+            - name: tensorflow
+              image: nickharvey/pach_kflow_example:v1
+              env:
+                # This endpoint assumes that the pachd service was deployed
+                # in the namespace pachyderm.
+                # You may replace this with pachd.<namespace> if you deployed
+                # pachyderm in another namespace. For example, if deployed
+                # in default it would be pachd.default. You may also
+                # hard code in the pachd CLUSTER-IP address you obtain from
+                # kubectl get services -n <namespace>
+                - name: S3_ENDPOINT
+                  value: "pachd.pachyderm:600"
+                - name: S3_USE_HTTPS
+                  value: "0"
+                - name: S3_VERIFY_SSL
+                  value: "0"
+              command:
+                - "python3"
+                - "tfjob.py"
+                - "-i"
+                - "master.inputrepo"
+                - "-o"
+                - "master.outputrepo"
+                - "-d"
+                - "/tmp/data/"
+```
+The 'mnist_tf_job_s3_gateway.yaml' is our spec file that Kubeflow and Kubernetes will use to deploy our code. You can find out everything you need to know about this spec file in the [Kubeflow TFjobs Docs](https://www.kubeflow.org/docs/components/training/tftraining/). Notice the Pachyderm repos, branches, and tmp location are being declared at the bottom.  
+
+To deploy just run the following:
+`kubectl create -f mnist_tf_job_s3_gateway.yaml`
+
+#### A few moments later...  
+
+We can check on things by going to the  and click on TFJob Dasboard. You should see something like:
+
+![tfjob-dashboard](tfjob-dashboard.png)
+
+## Step 4 - Trust but verify
+You know the old saying, always trust but verify too. Let's confirm that we actually  trained our model and that we maintained data provenance as we worked our way down the pipeline. 
+
+```
+➜ pachctl list file outputrepo@master:/data/ --history all
+COMMIT                           NAME              TYPE COMMITTED   SIZE     
+a0f654c7a65f42f69e8bddd1a2035a7e /data/mnist.npz   file 7 hours ago 10.96MiB 
+eb7e51147b4d42cfbcd555c84be778d2 /data/my_model.h5 file 7 hours ago 4.684MiB 
+```
+Perfect, the data is exactly where wanted it to go. Now, lets see how Pachyderm can show us where it came from before it got moved. Simply run the following:
+
+```
+➜ pachctl inspect commit outputrepo@eb7e51147b4d42cfbcd555c84be778d2
+Commit: outputrepo@eb7e51147b4d42cfbcd555c84be778d2
+Original Branch: master
+Parent: a0f654c7a65f42f69e8bddd1a2035a7e
+Started: 7 hours ago
+Finished: 7 hours ago
+Size: 15.64MiB
+```
+
+How cool is that? Our `my_model.h5` (commit: eb7e51147b4d42cfbcd555c84be778d2) has a _parent_ of `a0f654c7a65f42f69e8bddd1a2035a7e` which is the commit id of `mnist.npz` file we checked in earlier.  
+
+Now when anyone asks "What data was used to train that model?" you can tell them with just one command.
```

Comment:
Now, (add a comma)
Now when anyone asks "What data was used to train that model?", (add a comma)

Overall, I think this sentence is too informal. You can change to something like this:

The `pachctl inspect command` enables you to track the origin of any commit in your output repository.